lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -22,6 +22,8 @@ const Runner = require('../../../lighthouse-core/runner');
 const Config = require('../../../lighthouse-core/config');
 const configJSON = require('../../../lighthouse-core/config/default.json');
 const log = require('../../../lighthouse-core/lib/log');
+const STORAGE_KEY = 'lighthouse_audits';
+const _flatten = arr => [].concat.apply([], arr);
```

Comment:
Would it be worth putting _flatten in a util.js which get required? I declare it 2 times. I don't think it's worth but what do you guys think? (@brendankenny, @paulirish)